mkdocs/livereload/__init__.py

```diff
@@ -86,6 +86,21 @@ def callback(event):
         handler.on_any_event = callback
         self.observer.schedule(handler, path, recursive=recursive)
```

Comment:
Shouldn't this line be removed as we are now doing this down below?

Reply:
no (no to remove, no its not doing the same below). this is the initial schedule and the main operation.

good question thought, after adding schedule I removed the following lines directly following, so it got lost in transpositions and your question made me aware that change 1. got completely under the wheels.

~~~
if not path == os.path.realpath(path):
    self.observer.schedule(handler, os.path.realpath(path), recursive=recursive)
~~~

if my concentration is still well enough this should be edited in.